pkg/driverbuilder/templates.go

```diff
@@ -64,6 +69,27 @@ func renderMakefile(w io.Writer, md makefileData) error {
 	return t.Execute(w, md)
 }
 
+func LoadMakefileObjList(c builder.Config) (string, error) {
```

Comment:
Get the raw content of makefile.in in libs repo at requested driverversion;
then, find the `@DRIVER_NAME@-y +=` row and take the second part (ie: the list of objects required for the module).